rasa/core/channels/mattermost.py

```diff
@@ -86,27 +216,21 @@ def blueprint(self, on_new_message):
         @mattermost_webhook.route("/webhook", methods=["POST"])
         async def webhook(request: Request):
             output = request.json
-            if output:
-                # splitting to get rid of the @botmention
-                # trigger we are using for this
-                text = output["text"].split(" ", 1)
-                text = text[1]
-                sender_id = output["user_id"]
-                self.bot_channel = output["channel_id"]
-                try:
-                    out_channel = MattermostBot(
-                        self.url, self.team, self.user, self.pw, self.bot_channel
-                    )
-                    user_msg = UserMessage(
-                        text, out_channel, sender_id, input_channel=self.name()
-                    )
-                    await on_new_message(user_msg)
-                except Exception as e:
-                    logger.error(
-                        "Exception when trying to handle message.{0}".format(e)
-                    )
-                    logger.debug(e, exc_info=True)
-                    pass
-            return response.text("")
+
+            if not output:
+                return response.text("")
+
+            sender_id = output["user_id"]
```

Comment:
```suggestion
            
```
Can be removed as it is never used. Reason for travis failure.

Reply:
True, leftovers from the refactoring...